src/statisticBlock.js

```diff
@@ -468,8 +470,14 @@ class StatisticBlock extends Component {
 
   onDimensionLabelClick(dimNo, value) {
     const { services } = this.props;
+    const component = services.QlikComponent;
+    const inStoryMode = component.options.selections;
```

Comment:
if we're making a check a line below for `services.QlikComponent`, we could have this check under line 475, just before we use it.